README.md

```diff
@@ -61,6 +61,7 @@ module.exports = {
       location: '@vue-storefront/spree-api/server',
       configuration: {
         backendUrl: process.env.BACKEND_URL,
+        assetsURL: process.env.ASSETS_URL,
```

Comment:
We should be consistent and either change this one to `assetsUrl` or rename other urls to something like `backendURL`